pkg/httpserver/http_server.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	MIMEApplicationVendorIOTASerializerV1 = "application/vnd.iota.serializer-v1"
+	MIMEApplicationVendorIOTASerializerV2 = "application/vnd.iota.serializer-v2"
```

Comment:
Why don't you add a new constant instead? what if we want to use inx-app for older formats?